staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/conversion.go

```diff
@@ -63,7 +63,7 @@ func ConvertJSONSchemaPropsToOpenAPIv2Schema(in *apiextensions.JSONSchemaProps)
 		}
 
 		switch {
-		case len(p.Type) == 2 && (p.Type[0] == "null" || p.Type[1] == "null"):
```

Comment:
Don't we need to keep this case for aggregating pre-1.15 apiserver openapi?

Reply:
forgot to set nullable to false. It's basically the same as what we had before, only without going through a type slice with `null` and the type.

Reply:
If this is all internal, that's fine. I thought part of this was aggregating things read from aggregated servers (though I guess that would be reading v2, which has neither nullable nor slice type)